File: benchmark/results/ts-script-sonnet/run-001/b-baseline/output/src/domain/notification/services/NotificationRepository.ts
import { Notification } from '../aggregates/Notification';

/** NotificationRepository service. */
export class NotificationRepository {
  create(input: Partial<Notification>): Notification { throw new Error('not implemented'); }
  findById(id: string): Notification | null { throw new Error('not implemented'); }
  list(limit: number): Notification[] { throw new Error('not implemented'); }
  delete(id: string): void { throw new Error('not implemented'); }
}

File: benchmark/results/python-multilang/run-001/a-mcp/output/notification/services/notification_repository.py
from notification.aggregates.notification import Notification

"""NotificationRepository service."""
class NotificationRepository:
    def create(self, input: dict) -> Notification:
        raise NotImplementedError('not implemented')
    def find_by_id(self, id: str) -> Notification | None:
        raise NotImplementedError('not implemented')
    def list(self, limit: int) -> list[Notification]:
        raise NotImplementedError('not implemented')
    def delete(self, id: str) -> None:
        raise NotImplementedError('not implemented')

